apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/pagerules/PageRuleServiceImpl.java

```diff
@@ -817,4 +823,50 @@ public ViewRuleResponse getRuleResponse(Long ruleId) {
                 ruleMetadata.getRuleDescText(), ruleMetadata.getSourceQuestionIdentifier(), sourceValues,
                 ruleMetadata.getLogic(), ruleMetadata.getTargetType(), ruleMetadata.getErrormsgText(), targetValues);
     }
+
+    @Override
+    public Page<ViewRuleResponse> getAllPageRule(Pageable pageRequest) {
```

Comment:
This class is already very large, these new methods have their own concerns and can move to separate classes.